src/main/java/com/mhrglobal/payment/BankPaymentService.java

```diff
@@ -1,17 +1,22 @@
 package com.mhrglobal.payment;
 
+import com.mhrglobal.clients.PaymentClient;
+import com.mhrglobal.domain.Employee;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-//Do not change
```

Comment:
I needed to change this implementation and the interface to be able to pass an Employee.